lib/commands/createcache.js

```diff
@@ -77,15 +77,11 @@ function createCache(opts, request, done){
   
   if(opts.cacheExpiryByDate){
     createCachePayload.expirySettings.expiryDate.value = opts.cacheExpiryByDate;
-  } else if(opts.cacheExpiryInSecs){
-    var expirySettings = {
-      "timeoutInSec": {
-        "value": opts.cacheExpiryInSecs
-      },
-      "valuesNull": false
-    }
-    createCachePayload.expirySettings = expirySettings;
-  }     
+  }
+  if(opts.cacheExpiryInSecs){
+    createCachePayload.expirySettings.timeoutInSec = {};
```

Comment:
Last nit, why not just do this in one line instead of two?  You could use the following to do it in one statement:

```
createCachePayload.expirySettings.timeoutInSec = {
  value: opts.cacheExpiryInSecs
};
```

Reply:
PR is updated to use one line for expiry settings.